go/grpcweb/websocket_wrapper.go

```diff
@@ -39,6 +72,9 @@ func (w *webSocketResponseWriter) Write(b []byte) (int, error) {
 	if !w.writtenHeaders {
 		w.WriteHeader(http.StatusOK)
 	}
+	if w.timeOutInterval > time.Second && w.timer != nil {
+		w.timer.Reset(w.timeOutInterval)
+	}
```

Comment:
This just becomes a matter of setting `lastMessageTime`
```suggestion
	w.tmu.Lock()
	w.lastMessageTime = time.Now()
	w.tmu.Unlock()
```